.dotnet/src/Custom/Audio/AudioTranscriptionFormat.cs

```diff
@@ -1,41 +1,48 @@
+using System;
 using System.ComponentModel;
 
 namespace OpenAI.Audio;
 
-// CUSTOM: Renamed.
+// CUSTOM: Introduced as scenario-specific redirection of common format.
 /// <summary> The format of the transcription. </summary>
-[CodeGenModel("CreateTranscriptionRequestResponseFormat1")]
-public readonly partial struct AudioTranscriptionFormat
+public readonly partial struct AudioTranscriptionFormat : IEquatable<AudioTranscriptionFormat>
 {
-    // CUSTOM:
-    // - Applied the EditorBrowsable attribute.
-    // - Added custom doc comments.
+    internal readonly InternalAudioResponseFormat _internalAudioResponseFormat;
```

Comment:
I can give that a shot. My concern is that we're growing our spec pollution debt (convoluting .tsp for emitter purposes) and there's increasing urgency/pressure/value for us to use this spec and an upgraded Azure counterpart to generate the Swagger documents and replace AOAI's current handwritten ones. This'll be a particularly egregious case since we'll end up with *three* copies of the same thing in the .yaml.

Still probably the right thing to do right now, though, and likely addressable via something holistic like using a different `client.tsp` for .cs emission and keeping all the pollution contained in that.

Reply:
Updated -- we can eliminate the extra tool behavior once we have clarity on .tsp enum use and/or union spread support.